src/functions.php

```diff
@@ -19,13 +27,17 @@
 use Prooph\EventStore\Projection\ProjectionManager;
 use Prooph\EventStore\StreamName;
 use ProophTest\EventStore\Mock\TestDomainEvent;
+use Prooph\EventStore\ArangoDb\Projection\ProjectionManager as ArangoDbProjectionManager;
+use Prooph\EventStore\ArangoDb\EventStore as ArangoDbEventStore;
+use Prooph\EventStore\ArangoDb\PersistenceStrategy\AggregateStreamStrategy as ArangoDbAggregateStreamStrategy;
 
 function testDatabases(): array
 {
     return [
         'mysql' => getenv('MYSQL_DB'),
         'mariadb' => getenv('MARIADB_DB'),
         'postgres' => getenv('POSTGRES_DB'),
+        'arangodb' => getenv('ARANGODB_DBNAME'),
```

Comment:
I used DB for all other drivers, not DBNAME. Let's make this consistent, either DB or DBNAME, you can chose :)

Reply:
Done